CUSTOMIZATION.md

```diff
@@ -99,6 +99,9 @@ module Kaui
   # Link to the customer invoice (HTML, PDF, etc.)
   self.customer_invoice_link = lambda { |invoice, ctx| ctx.link_to 'View customer invoice html', ctx.kaui_engine.show_html_invoice_path(invoice.invoice_id), :class => 'btn', :target => '_blank' }
 
+  # Additinal Link to the customer invoice
+  self.additional_invoice_links = lambda { |invoice, ctx| ctx.link_to 'Addtinal invoice link', ctx.metal_engine.bulk_invoice_adjustment_path(invoice.account_id, invoice.invoice_id), class: 'btn', target: '_self' }
```

Comment:
The example is not very practical for users (`metal_engine` is proprietary). What does `ctx.metal_engine.bulk_invoice_adjustment_path` return? A link (maybe wrapped in a `<div>`)? Could you just hardcode that string?